_mobile/android.md

```diff
@@ -244,41 +248,38 @@ org.pytorch.Tensor
 
 If the reader is familiar with pytorch python api, we can think that org.pytorch.Tensor represents torch.tensor, org.pytorch.Module torch.Module<?>, while org.pytorch.IValue represents value of TorchScript variable, supporting all its types. ( https://pytorch.org/docs/stable/jit.html#types )
```

Comment:
- Capitalization "pytorch python api" --> "PyTorch Python API"
- Remove the <?> after Module
- Remove spaces before and after parenthesis around the url

Revision: 
"If the reader is familiar with [the PyTorch Python API], we can think [of] org.pytorch.Tensor [representing] torch.tensor, org.pytorch.Module [representing] torch.Module, [and] org.pytorch.IValue [representing the] of TorchScript [the] variable, supporting all its [types](https://pytorch.org/docs/stable/jit.html#types).